packages/grammar-selector/lib/grammar-list-view.js

```diff
@@ -20,7 +20,7 @@ class GrammarListView {
         const div = document.createElement('div')
         div.classList.add('pull-right')
 
-        if (grammar.constructor.name === "TreeSitterGrammar") {
+        if (isTreeSitter(grammar)) {
```

Comment:
This is used a few times, and as a function adds better separation of the idea from the implementation